libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -262,6 +262,92 @@ const AP_Param::GroupInfo AR_AttitudeControl::var_info[] = {
     // @User: Standard
     AP_SUBGROUPINFO(_sailboat_heel_pid, "_SAIL_", 12, AR_AttitudeControl, AC_PID),
 
+    // @Param: _ROLL_P
+    // @DisplayName: Roll control P gain
+    // @Description: Roll angle controller P gain
+    // @Range: 0.000 2.000
+    // @Increment: 0.01
+    // @User: Standard
+
+    // @Param: _ROLL_I
+    // @DisplayName: Roll control I gain
+    // @Description: Roll angle controller I gain
+    // @Range: 0.000 2.000
+    // @User: Standard
+
+    // @Param: _ROLL_IMAX
+    // @DisplayName: Roll control I gain maximum
+    // @Description: Roll control I gain maximum gain
+    // @Range: 0.000 1.000
+    // @Increment: 0.01
+    // @User: Standard
+
+    // @Param: _ROLL_D
+    // @DisplayName: Roll control D gain
+    // @Description: Roll control D gain
+    // @Range: 0.000 0.100
+    // @Increment: 0.001
+    // @User: Standard
+
+    // @Param: _ROLL_FF
+    // @DisplayName: Roll control feed forward
+    // @Description: Roll control feed forward
+    // @Range: 0.000 0.500
+    // @Increment: 0.001
+    // @User: Standard
+
+    // @Param: _ROLL_FILT
+    // @DisplayName: Roll control filter frequency
+    // @Description: Roll control input filter.  Lower values reduce noise but add delay.
+    // @Range: 0.000 100.000
+    // @Increment: 0.1
+    // @Units: Hz
+    // @User: Standard
+    AP_SUBGROUPINFO(_roll_pid, "_ROLL_", 13, AR_AttitudeControl, AC_PID),
+
+    // @Param: _PITCH_P
+    // @DisplayName: Pitch control P gain
+    // @Description: Pitch angle controller P gain
+    // @Range: 0.000 2.000
+    // @Increment: 0.01
+    // @User: Standard
```

Comment:
The increment value is not in the setting range.
I think that it is better to put together.